site/_posts/2017-09-11-changelogs.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Last Week in ManageIQ: Why Changelogs - Cause all the Cool Kids Do It"
+author: jprause
+date: 2017-09-12
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi all,...[John Prause](https://github.com/jprause) here. I'm usually the one behind-the-scenes, but I thought I'd take a shot at writing one of these, so here goes...
+
+First, some statistics about the blog entries for Last Week in ManageIQ:
+  - The [first blog entry](http://manageiq.org/blog/2016/07/last-week-in-manageiq-faster-reports-cloud-networks-and-more/) was on July 26, 2016
+  - We’ve had 54 blog posts
+  - 25 distinct authors
+  - Four authors have written 5 blog entries each
+  - We’ve had 22 Sprint Reviews since the first blog entry
+  - Each blog submission is peer-reviewed by 4-5 other prior bloggers
+  - We’ve also had a [video blog](http://manageiq.org/blog/2017/08/limericks-week-in-miq/) submission
+  - Did you know that YOU can also submit a blog entry,..after all, we are open source
+
+Usually these blog entries review the PRs from last week. However, I’d like to talk about an important aspect of our [ManageIQ repos](https://github.com/ManageIQ/),..and that is, changelogs. According to [keepachangelog.com][keepachangelog], “A changelog is a file which contains a curated, chronologically ordered list of notable changes for each version of a project.”
+
+Here at ManageIQ we follow the guiding principles and format as described at [keepachangelog.com][keepachangelog] This is no easy task and takes a concerted effort,...but the end result is meant to help you, the community user of ManageIQ.
```

Comment:
Need a period after the keepachangelog link.

Reply:
Also remove the elipsis

Reply:
Thanks @Fryguy for catching the AWOL period. Will update.
Yeah, the ,... are kind of my expressive writing style. Thanks!